nomenclature/definitions/variable/technology/power-plant.yaml

```diff
@@ -464,12 +630,45 @@ Inertia|Electricity|Hydro|Reservoir:
 Rate Frequency Containment Reserve|Electricity|Hydro|Reservoir:
    description: Percentage of the Maximum Active Power that can be devoted to Frequency Containment Reserve
                 for a typical "simple Hydro reservoir " unit
-   unit: '%'
+   unit: "%"
 
 Rate Automatic Frequency Restoration Reserve|Electricity|Hydro|Reservoir:
    description: Percentage of the Maximum Active Power that can be devoted to Automatic Frequency Restoration Reserve
                 for a typical "simple Hydro reservoir " unit
-   unit: '%'
+   unit: "%"
+
+# description of Energy Storage Systems
+Maximum Energy Charge|Electricity|Energy Storage System:
+   description: This is the upper bound that energy storage system could charge energy at time "t".
+   unit: GWh
+
+Initial Storage Level|Electricity|Energy Storage System:
+   description: Initial energy stored in the energy storage system unit at first load level. (ESS that includes hydro, battery, etc.)
+   unit: GWh
+
+Maximum Charge|Electricity|Energy Storage System:
+   description: Maximum charge when storing energy the energy storage system unit.
+   unit: MW
```

Comment:
Related to the conversation at https://github.com/openENTRANCE/nomenclature/issues/50#issuecomment-642560593 I suppose to add **Maximum Discharge**.

Reply:
Ok, got it.

Reply:
Perfect, thank you @erikfilias